test/t_new_func.rb

```diff
@@ -365,7 +365,18 @@ def test_check_tc_flag
 
     start_roma 'cpdbtest/config4cpdb_tc.rb'
     stop_roma
-    res = `#{bin_dir}/check_tc_flag --storage ./localhost_11211/roma --library /usr/local/roma/libexec`
+
+    # If `tchmgr` is system-widley known, like installation from `apt-get`,
+    # set the library path as it's parent directory.
+    #
+    # If not, assume it is under the libexec directory.
+    path_tchmgr = `which tchmgr`
+    libpath = Pathname.new(path_tchmgr).dirname.each_filename.to_a[0..-2]
+              .join(File::Separator).prepend(File::Separator) if "" != path_tchmgr
```

Comment:
umm, this line looks unreadable to me. 
This line attempts to extract the path to the directory in which `bin/tchmgr` are, doesn't it?
It can be done with `File.expand_path(File.join(path_tchmgr, '..', '..'))`.